aerleon/lib/policy_builder.py

```diff
@@ -34,10 +34,126 @@
 else:
     from typing import Annotated
 
+if sys.version_info < (3, 10):
+    from typing_extensions import TypeAlias
+else:
+    from typing import TypeAlias
+
+if sys.version_info < (3, 11):
+    from typing_extensions import NotRequired, Required
+else:
+    from typing import NotRequired, Required
+
+if sys.version_info < (3, 8):
```

Comment:
Can order these by their versions?

Reply:
There is probably a tidy way to express this kind of thing. But ordering by version would be an improvement.